superset/views/core.py

```diff
@@ -1811,7 +1810,7 @@ def dashboard(  # pylint: disable=too-many-locals
                 datasource = ConnectorRegistry.get_datasource(
                     datasource_type=datasource["type"],
                     datasource_id=datasource["id"],
-                    session=session,
+                    session=db.session(),
```

Comment:
Could you explain why is this change necessary?

Reply:
bc the session = db.session() was moved into the `Dashboard.get method`